### ISSUE TITLE
Allow wild cards in 'pa_dirs' config option

### ISSUE THREAD

**Issue by alanz** [erlang-ls/erlang_ls#407]
**Is your feature request related to a problem? Please describe.**

Once #397 lands, we will have a `pa_dirs` option in the config file. At the moment we need a line per library used.

**Describe the solution you'd like**

Allow the directories to include wild cards, for example

     - "_build/default/lib/*/ebin"

**Comment by alanz** [comment]
We took a different approach, did not land #397, this issue is no longer relevant.
